packages/core-data/src/resolvers.js

```diff
@@ -118,3 +119,18 @@ export function* hasUploadPermissions() {
 
 	yield receiveUploadPermissions( includes( allowHeader, 'POST' ) );
 }
+
+/**
+ * Request autosave data from the REST API.
+ *
+ * @param {string} postType The type of the parent post.
+ * @param {number} postId   The id of the parent post.
+ */
+export function* getAutosave( postType, postId ) {
+	const { baseURL } = yield select( 'getEntity', 'postType', postType );
```

Comment:
you could use a `resolveSelect( 'getPostType`, postType )` control like what I have in [this pull](https://github.com/WordPress/gutenberg/pull/13716/files#diff-e66041d992a60b9b29e5b80f84d7b612R23) and that should work.

Reply:
> you could use a `resolveSelect( 'getPostType', postType )` control like what I have in [this pull](https://github.com/WordPress/gutenberg/pull/13716/files#diff-e66041d992a60b9b29e5b80f84d7b612R23) and that should work.

@talldan Did you try this? It's not unexpected that `select` might return `undefined` if it hasn't yet been resolved.

Reply:
I've made that change now. 

I suppose `yield select( 'getEntity', 'postType', postType );` always worked because of preloading? In that case it definitely makes sense to use resolveSelect 👍